docs/infrastructure/workers/worker-tools-versioning-and-caching.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Worker Tools, Versioning and Caching
+description: Worker Tools 
+position: 50
+---
+
+Worker Tools are a set of docker images used as [execution containers for workers](https://octopus.com/docs/projects/steps/execution-containers-for-workers)] to run deployment processes. Worker Tools include a wide range of software tools with the aim of supporting a majority of deployment scenarios out of the box. This post focuses on how we create these Worker Tool images, version, cache on workers and release them.
+
+## Versioning Worker Tools
+Worker Tool images, follow a semantic versioning (SemVer) approach of `Major.Minor.Patch-Distro` for their tag format. When we release a new version of Worker Tools to the [Worker Tools dockerhub repository](https://hub.docker.com/r/octopusdeploy/worker-tools/tags) we also add the following image tags, distribution (`ubuntu.18.04` or `windows.ltsc2019`), `Major-Distro` and `Major.Minor-Distro`. We recommend using the fully qualified SemVer as patch updates of Worker Tools could result in an updated tool depenedncy introducing a breaking change.
```

Comment:
For `Major-Distro` and so on, perhaps provide an "`(e.g. )`" to make it easier to understand?